### DISCUSSION TITLE
The cron command cannot be run

### DISCUSSION THREAD

**Discussion by hqy662** [termux/termux-packages#17759] [General]
crond runs in the foreground. 
crontab -l
*/1 * * * * echo hello>>~/t.txt

**Comment by sylirre** [comment]
Your issue is not reproducible. The used steps:

* `pkg install cronie`
* `crond`
* `crontab -e`

Crontab contents:
```
*/1 * * * * echo hello >> ~/t.tx
```

logcat entry:
```
09-04 22:12:01.315 10949 10949 I CROND   : (u0_a290) CMDEND (echo hello >> ~/t.tx)
```

File `~/t.tx` is created within a minute as should be.

***

If you want to have `crond` automatically started as service, use https://wiki.termux.com/wiki/Termux-services.

`crond` works when Termux app kept opened and wake lock is enabled.

If wake lock is disabled, the service execution will slow down or be freezed when screen is off. If Termux app is closed, the Android is free to kill all its remaining processes. This behavior is not a bug.

**Reply by hqy662** [reply]
Cronie has been installed. Logcat doesn't show crond.

**Reply by sylirre** [reply]
Because `crond` has to be launched manually. It won't start itself. That's why you don't see it in a logcat.

**Reply by hqy662** [reply]
Crond is already running, ps can see it.
